### ISSUE TITLE
Series.reorder_levels docstring includes extra `axis` argument.

### ISSUE THREAD

**Issue by tschm** [pandas-dev/pandas#22627]
#### Code Sample, a copy-pastable example if possible

```python

    def reorder_levels(self, order):
        """
        Rearrange index levels using input order. May not drop or duplicate
        levels

        Parameters
        ----------
        order : list of int representing new level order.
               (reference level by number or key)
        axis : where to reorder levels

        Returns
        -------
        type of caller (new object)
        """
        if not isinstance(self.index, MultiIndex):  # pragma: no cover
            raise Exception('Can only reorder levels on a hierarchical axis.')

        result = self.copy()
        result.index = result.index.reorder_levels(order)
        return result


```
#### Problem description

[this should explain **why** the current behaviour is a problem and why the expected output is a better solution.]

**Note**: We receive a lot of issues on our GitHub tracker, so it is very possible that your issue has been posted before. Please check first before submitting so that we do not have to handle and close duplicates!

**Note**: Many problems can be resolved by simply upgrading `pandas` to the latest version. Before submitting, please check if that solution works for you. If possible, you may want to check if `master` addresses this issue, but that is not necessary.

For documentation-related issues, you can check the latest versions of the docs on `master` here:

https://pandas-docs.github.io/pandas-docs-travis/

If the issue has not been resolved there, go ahead and file it in the issue tracker.

#### Output of ``pd.show_versions()``

<details>

[paste the output of ``pd.show_versions()`` here below this line]

</details>

**Comment by TomAugspurger** [comment]
We can update the docs, or add an unused `axis` to Series.reorder_levels.

If we decide to add `axis` to Series.reorder_levels, it'll probably be easiest to just move the implementation to `NDFrame.reorder_levels` and use `self._get_axis(axis)`

**Comment by jbrockmendel** [comment]
Closed by #22734
